gcl_sdk/audit/dm/models.py

```diff
@@ -0,0 +1,82 @@
+import datetime
+import inspect
+
+from restalchemy.common.contexts import ContextIsNotExistsInStorage
+from restalchemy.common.contexts import get_context
+from restalchemy.dm import models
+from restalchemy.dm import properties
+from restalchemy.dm import types
+from restalchemy.storage.sql import orm
+
+from gcl_sdk.audit import constants
+
+
+class AuditRecord(
```

Comment:
Do I understand correctly that user should get it's logs by project? And get logs of another project should be restricted?

In this case we need project_id field in model (see other examples where we use it, ask if not found).